tests/cpp/common/test_transform_range.cc

```diff
@@ -50,13 +50,26 @@ TEST(Transform, DeclareUnifiedTest(Basic)) {
   out_vec.Fill(0);
 
   Transform<>::Init(TestTransformRange<bst_float>{},
-	                Range{0, static_cast<Range::DifferenceType>(size)},
-	                TRANSFORM_GPU)
+                    Range{0, static_cast<Range::DifferenceType>(size)},
+                    TRANSFORM_GPU)
       .Eval(&out_vec, &in_vec);
   std::vector<bst_float> res = out_vec.HostVector();
 
   ASSERT_TRUE(std::equal(h_sol.begin(), h_sol.end(), res.begin()));
 }
 
+#if !defined(__CUDACC__)
```

Comment:
Why is this macro guard needed?

Reply:
@hcho3 This file is included in another cu file so we can have test in both gpu and cpu. (that's what the transform for, define one function and run it on both sides).  But this test only runs on CPU since it's related to openmp, so we don't compile it on cuda mode.